### PR TITLE
Add installer

### PR DESCRIPTION
**Pull request by nhynes**
This allows users to `cargo install oasis-chain`. This is like getting around publishing the `.tex` on arxiv by `\include`ing the built pdf.

### CODE REVIEW

**Comment by peterjgilbert** [comment]
Let's add a build step to CI. Otherwise looks good.

File: installer/build.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    let rustflags = envs.entry("RUSTFLAGS".to_string()).or_default();
+    *rustflags = format!("{} -Ctarget-feature=+aes,+ssse3", rustflags);
+    std::process::Command::new("cargo")
+        .args(&[
+            "install",
+            "--force",
+            "--git",
+            "https://github.com/oasislabs/oasis-chain",
+            "--tag",
+            "v0.1.0",
```
Comment (peterjgilbert) [inline installer/build.rs]:
```suggestion
            "v0.1.0",
            "--debug",
```

Comment (nhynes) [reply installer/build.rs]: why `--debug`?

Comment (peterjgilbert) [reply installer/build.rs]: Printing stdout/stdout in parity is behind `debug_assertions` flag. Perhaps we just change that to a feature flag.

**Pull request merged**